<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThreeQuery</class>
 <widget class="QWidget" name="ThreeQuery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="Com_ThreeQuery_lbL1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Com_ThreeQuery_leR1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>60</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Com_ThreeQuery_lbL2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Com_ThreeQuery_leR2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Com_ThreeQuery_lbL3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Com_ThreeQuery_leR3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Com_ThreeQuery_lbTitle">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Com_ThreeQuery_btConfirm">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Com_ThreeQuery_btCancel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>260</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
